src/main/java/com/mnewt00/vulcandatabase/storage/SQLiteStorageProvider.java

```diff
@@ -0,0 +1,127 @@
+package com.mnewt00.vulcandatabase.storage;
+
+import com.google.common.collect.Lists;
+import com.mnewt00.vulcandatabase.Log;
+import com.mnewt00.vulcandatabase.VulcanDatabase;
+import lombok.Getter;
+import lombok.SneakyThrows;
+
+import java.io.File;
+import java.sql.*;
+import java.util.List;
+import java.util.UUID;
+
+public class SQLiteStorageProvider implements StorageProvider {
+
+    @Getter
+    private final Connection connection;
+
+    public SQLiteStorageProvider() {
+        Connection connection1;
```

Comment:
Using a more friendly name rather than `connection1` would be great, for example.